bin/improver-threshold

```diff
@@ -131,12 +164,38 @@ def main():
         thresholds = args.threshold_values
         fuzzy_bounds = None
 
-    result = BasicThreshold(
+    result_no_realization = BasicThreshold(
```

Comment:
We might want to change this variable name, as I think this variable has a `realization` coordinate, as it hasn't yet been collapsed.